package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "channelape-sdk",
-	"version": "1.1.0",
+	"version": "1.1.0-develop.1",
```

Comment:
I would consider this a new minor release since we are adding the ability for the SDK to work within a browser. Please change this to 1.2.0-develop.1